manifests/e/ente-io/auth-desktop/3.0.4/ente-io.auth-desktop.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: ente-io.auth-desktop
+PackageVersion: 3.0.4
+
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Scope: machine
+InstallerSwitches: 
+  Silent: /VERYSILENT /SUPPRESSMSGBOXES
+  SilentWithProgress: /SILENT /SUPPRESSMSGBOXES

```

Comment:
```suggestion
```
These shouldn't be necessary, as they are the default switches for inno installers

Reply:
aight